test/cppsuite/test_harness/workload/thread_context.cpp

```diff
@@ -203,14 +203,12 @@ thread_context::finish()
 }
 
 std::string
-thread_context::key_to_string(uint64_t key_id)
+thread_context::left_padding_to_string(uint64_t value, uint64_t size)
```

Comment:
This name is a bit ambiguous. Consider `int_to_string_with_padding`

Reply:
I renamed it to `string_with_padding` as I changed the first arg type too, see 08e65c521a2baed13044cba9eef80fc905973989